plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/ConfigurationCacheTest.kt

```diff
@@ -10,22 +10,11 @@ import org.jlleitschuh.gradle.ktlint.testdsl.build
 import org.jlleitschuh.gradle.ktlint.testdsl.project
 import org.junit.jupiter.api.DisplayName
 
-@GradleTestVersions(minVersion = "6.6.1")
+@GradleTestVersions
 class ConfigurationCacheTest : AbstractPluginTest() {
     private val configurationCacheFlag = "--configuration-cache"
     private val configurationCacheWarnFlag = "--configuration-cache-problems=warn"
 
-    /**
-     * 2 warnings are still reported by the Kotlin plugin. We can't fix them.
-     * But make sure we aren't creating more issues.
-     * ```
-     * 2 problems were found storing the configuration cache, 1 of which seems unique.
-     * plugin 'org.jetbrains.kotlin.jvm': registration of listener on 'Gradle.addBuildListener' is unsupported
-     * See https://docs.gradle.org/6.6-milestone-3/userguide/configuration_cache.html#config_cache:requirements:build_listeners
-     * ```
-     */
-    private val maxProblemsFlag = "-Dorg.gradle.unsafe.configuration-cache.max-problems=2"
```

Comment:
I am not sure if it is still applicable, I run the tests with the value set to 0 and everything worked correctly.

Reply:
Do we want to restore and set the value to 0?